modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,162 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="replacing-a-bare-metal-control-plane-node_{context}"]
+= Replacing a bare-metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} cluster.
+
+[IMPORTANT]
+====
+When the replacement BareMetalHost is created from a new physical machine or an incomplete prior YAML file configuration, the `externallyProvisioned:` field in the configuration file must be set to `false` to allow the cluster to provision the host. 
+
+Existing control plane BareMetalHosts may have the externallyProvisioned flag set to true if they were provisioned by {product-title}.
+====
+
+.Procedure
+
+. Create the `baremetalhost` and `ipmi` secret:
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+----
```

Comment:
In 4.10 with virtual media, the bmh file look like:
```
apiVersion: metal3.io/v1alpha1
kind: BareMetalHost
metadata:
  name: ocp-master-2
  namespace: openshift-machine-api
spec:
  automatedCleaningMode: disabled
  bmc:
    address: redfish-virtualmedia+http://192.168.124.113:8000/redfish/v1/Systems/f87eaf82-b32d-4291-ace6-b28677964e78
    credentialsName: ocp-master-2-bmc-secret
  bootMACAddress: aa:aa:aa:aa:ab:03
  bootMode: UEFI
  externallyProvisioned: false
  hardwareProfile: unknown
  online: true
  ```

Reply:
Updated as suggested.